packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -0,0 +1,762 @@
+.onfido-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.6rem;
+
+    &-view_wrapper {
+        position: relative;
+        min-width: 32.8rem;
+        min-height: 50rem;
+        margin: auto;
+    }
+
+    .account-form__poi-confirm-example_container {
+        margin-bottom: 0;
+    }
+
+    .onfido-sdk-ui-Modal-inner {
+        border-radius: 0.8rem;
+    }
+
+    @include mobile() {
+        padding: 1.6rem;
+
+        .onfido-sdk-ui-PageTitle-titleSpan {
+            font-size: 2rem;
+        }
+    }
+
+    .onfido-sdk-ui-Camera-webcamContainer {
+        left: auto;
+        width: -webkit-fill-available;
+    }
+
+    &__disabled {
+        opacity: 0.5;
+        pointer-events: none;
+    }
+
+    &__info-message {
+        position: absolute;
+        top: 2.4rem;
+        left: 25%;
+        z-index: 1;
+        @include mobile {
+            top: 1.6rem;
+            left: 3.1rem;
+        }
+    }
+
+    &__status-message {
+        background-color: var(--transparent-correct-message);
+        justify-content: flex-start;
+        transition: transform 0.35s ease, opacity 0.35s linear;
+        transform-origin: top;
+        opacity: 1;
+        width: 98%;
+
+        &--exit {
+            transform: scale(1, 0);
+            opacity: 0;
+        }
+
+        &_container {
+            position: absolute;
+            top: 0.4rem;
+            min-height: 3.4rem;
+            text-align: center;
+            z-index: 1;
+            width: 100%;
+        }
+    }
+}
+
+.idv-layout {
+    width: 100%;
+    .poi-form-on-signup__fields {
+        .proof-of-identity__container {
+            width: 100%;
+        }
+        .proof-of-identity__fieldset {
+            @include desktop {
+                margin-bottom: unset;
+            }
+            @include mobile {
+                margin-bottom: 2rem;
+            }
+            &-input {
+                @include desktop {
+                    margin-bottom: unset;
+                }
+            }
+        }
+    }
+    .proof-of-identity__inner-container {
+        @include desktop {
+            &--incl-image {
+                display: grid;
+                align-items: center;
+                column-gap: 1.5rem;
+                grid-template-columns: auto 0.5fr;
+            }
+        }
+    }
+}
+
+.idv-resubmit {
+    .poi-form-on-signup__fields {
+        .proof-of-identity__fieldset-container {
+            display: flex;
+            flex-direction: column;
+            gap: 3.2rem;
+        }
+    }
+}
+
+.upload-layout {
+    height: 100%;
+    overflow: auto;
+    .account-form__poi-confirm-example {
+        margin: unset !important;
+        margin-bottom: 3.2rem !important;
+    }
+}
+
+.account {
+    margin: 0 1.6rem;
+    &__scrollbars_container {
+        height: 100%;
+        padding-top: 2.4rem;
+        padding-bottom: 6.4rem;
```

Comment:
```suggestion
   padding: 2.4rem 0rem 6.4rem 
```

Reply:
@thisyahlen-deriv, all these CSS properties were created not by us, we just renamed the file and moved them to other.
I'm not sure if we should change them, if it is not related to our task.